subsys/net/lib/sockets/sockets.c

```diff
@@ -290,3 +349,32 @@ ssize_t zsock_recv(int sock, void *buf, size_t max_len, int flags)
 
 	return recv_len;
 }
+
+/* As this is limited function, we don't follow POSIX signature, with
+ * "..." instead of last arg.
+ */
+int zsock_fcntl(int sock, int cmd, int flags)
+{
+	struct net_context *ctx = INT_TO_POINTER(sock);
+
+	switch (cmd) {
+
```

Comment:
Sounds good, I just got to used that @tbursztyka asks to add more empty lines, not less ;-).